bootstrap.sh

```diff
@@ -47,10 +42,7 @@ else
     (cd zookeeper-$zk_ver/src/c && \
     ./configure --prefix=$zk_dist && \
     make install) && rm -rf zookeeper-$zk_ver zookeeper-$zk_ver.tar.gz)
-  if [ $? -ne 0 ]; then
-    echo "zookeeper build failed"
-    exit 1
-  fi
+  [ $? -eq 0 ] || "zookeeper build failed"
```

Comment:
Forgot the "fail". Also you could just do `...tar.gz) || fail "zookeeper build failed"`, but maybe that's more confusing.